generators/client/templates/angular/src/main/webapp/app/admin/audits/audits.component.ts.ejs

```diff
@@ -123,10 +123,6 @@ export class AuditsComponent implements OnInit {
 
     private onSuccess(audits: Audit[] | null, headers: HttpHeaders): void {
         this.totalItems = Number(headers.get('X-Total-Count'));
-        if (audits === null) {
-            this.audits = [];
-        } else {
-            this.audits = audits;
-        }
+        this.audits = audits ? audits : [];
```

Comment:
Simpler: `this.audits = audits || []`

Reply:
Thanks for the hint! I simplified this and other places. While doing these changes made some more cleaning:
* in `user-management.route` removed unnecessary `null` assignment
* in `no-pagination-template` removed assigning empty string to `this.currentSearch` because this is already empty string (if not empty then this code is never reached)